### PR TITLE
Release 2024.06.0

### PR DESCRIPTION
**Pull request by annejuan-okta**
## Description:
- **What's changed?**  Release PR for 2024.06.0
- **Is this PR related to a Monolith release?**  2024.06.0

### Resolves:

* [OKTA-726568](https://oktainc.atlassian.net/browse/OKTA-726568)

### CODE REVIEW

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx score

A minimum Acrolinx Score of **80** is required. The total score is an average of the subscores.

Select **Total score** to review the Acrolinx scorecard for your article. Try to increase your individual scores, for example: Correctness. Your content will be clearer and more consistent.

| Article | **Total score**<br />**Required:70** | Word and phrases<br />(Brand, terms)<br />Preferred: 80 | Correctness<br />(Spelling, grammar)<br />Preferred: 80 | Clarity<br />(Readability)<br />Preferred: 80 | Inclusive language<br />(+ accesibility)<br />Preferred: 80 |
|---------|:------------------------------------:|:--------------------------------------------------------:|:--------------------------------------------------------:|:---------------------------------------------:|:------------------------------------------------------------:|
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/concepts/inline-hooks/index.md](https://github.com/okta/okta-developer-docs/blob/e4c281ba7752688270bdd4b981145c71a3ecc14d/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fconcepts%2Finline-hooks%2Findex.md) | [83](https://okta.acrolinx.cloud/api/v1/checking/scorecards/c714d761-dae8-45d9-9a6b-a35f66fc042a) | 81  | 56  | 72  | 98  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/reference/hooks-best-practices/index.md](https://github.com/okta/okta-developer-docs/blob/e4c281ba7752688270bdd4b981145c71a3ecc14d/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Freference%2Fhooks-best-practices%2Findex.md) | [80](https://okta.acrolinx.cloud/api/v1/checking/scorecards/588981f7-03da-468a-86a8-601f6abe112c) | 85  | 57  | 85  | 100  |

  - Scorecard links, by default, are only good for 7 days. [Link for this commit's Content Analysis Report](https://okta.acrolinx.cloud/api/batch/gen.github.8befa16f-0aaf-45b0-87ca-bfe5dd06f05d?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

File: packages/@okta/vuepress-site/.vuepress/public/docs/okta-event-types.csv [hunk 1]
```diff
@@ -581,6 +584,7 @@ Event Type,Description,Release Date,Tags
...
+"security.breached_credential.detected","A credential, such as a password, which is associated with a known breach was used during an authentication flow. Used to identify users for whom credential rotation or other risk mitigation is necessary. The actor is the user with the breached credential. For OIE, a target will indicate the specificcredential associated with the breach. The outcome for this event will always be SUCCESS with a severity level of WARN. If breached credential protection is enabled, auser.session.clear will also be fired. These two events can be correlated by the Request ID.","2024.04.2","account","event-hook-eligible","security","user"
```
Comment (grahamsmith-okta) [inline packages/@okta/vuepress-site/.vuepress/public/docs/okta-event-types.csv:587]:
```suggestion
"security.breached_credential.detected","A credential, such as a password, which is associated with a known breach was used during an authentication flow. Used to identify users for whom credential rotation or other risk mitigation is necessary. The actor is the user with the breached credential. For Identity Engine, a target will indicate the specific credential associated with the breach. The outcome for this event will always be SUCCESS with a severity level of WARN. If breached credential protection is enabled, auser.session.clear will also be fired. These two events can be correlated by the Request ID.","2024.04.2","account","event-hook-eligible","security","user"
```

**Review by grahamsmith-okta — COMMENTED** [review]
Couple of suggestions

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx score

A minimum Acrolinx Score of **80** is required. The total score is an average of the subscores.

Select **Total score** to review the Acrolinx scorecard for your article. Try to increase your individual scores, for example: Correctness. Your content will be clearer and more consistent.

| Article | **Total score**<br />**Required:70** | Word and phrases<br />(Brand, terms)<br />Preferred: 80 | Correctness<br />(Spelling, grammar)<br />Preferred: 80 | Clarity<br />(Readability)<br />Preferred: 80 | Inclusive language<br />(+ accesibility)<br />Preferred: 80 |
|---------|:------------------------------------:|:--------------------------------------------------------:|:--------------------------------------------------------:|:---------------------------------------------:|:------------------------------------------------------------:|
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/concepts/inline-hooks/index.md](https://github.com/okta/okta-developer-docs/blob/9c9524261c616ebd609e56279d72b7e1cd338c4b/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fconcepts%2Finline-hooks%2Findex.md) | [83](https://okta.acrolinx.cloud/api/v1/checking/scorecards/6d85e231-47d6-4b5a-acdd-b466d67de045) | 81  | 56  | 72  | 98  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/reference/hooks-best-practices/index.md](https://github.com/okta/okta-developer-docs/blob/9c9524261c616ebd609e56279d72b7e1cd338c4b/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Freference%2Fhooks-best-practices%2Findex.md) | [80](https://okta.acrolinx.cloud/api/v1/checking/scorecards/67823429-e8b7-4171-b02c-b2b10dcb6824) | 85  | 57  | 85  | 100  |

  - Scorecard links, by default, are only good for 7 days. [Link for this commit's Content Analysis Report](https://okta.acrolinx.cloud/api/batch/gen.github.23eaab6c-8989-4cc7-86c8-e6fdded19341?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

**Review by grahamsmith-okta — DISMISSED** [review]

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx score

A minimum Acrolinx Score of **80** is required. The total score is an average of the subscores.

Select **Total score** to review the Acrolinx scorecard for your article. Try to increase your individual scores, for example: Correctness. Your content will be clearer and more consistent.

| Article | **Total score**<br />**Required:70** | Word and phrases<br />(Brand, terms)<br />Preferred: 80 | Correctness<br />(Spelling, grammar)<br />Preferred: 80 | Clarity<br />(Readability)<br />Preferred: 80 | Inclusive language<br />(+ accesibility)<br />Preferred: 80 |
|---------|:------------------------------------:|:--------------------------------------------------------:|:--------------------------------------------------------:|:---------------------------------------------:|:------------------------------------------------------------:|
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/concepts/inline-hooks/index.md](https://github.com/okta/okta-developer-docs/blob/12f7ef1d0137d2ec36a53414a021798b6657d132/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fconcepts%2Finline-hooks%2Findex.md) | [83](https://okta.acrolinx.cloud/api/v1/checking/scorecards/b8316a1a-26df-48c7-8ad2-cb9533829291) | 81  | 56  | 72  | 98  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/reference/hooks-best-practices/index.md](https://github.com/okta/okta-developer-docs/blob/12f7ef1d0137d2ec36a53414a021798b6657d132/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Freference%2Fhooks-best-practices%2Findex.md) | [80](https://okta.acrolinx.cloud/api/v1/checking/scorecards/025575ad-04c9-4872-9b3a-aaeda548fe00) | 85  | 57  | 85  | 100  |

  - Scorecard links, by default, are only good for 7 days. [Link for this commit's Content Analysis Report](https://okta.acrolinx.cloud/api/batch/gen.github.616a37ef-0023-4a79-ae46-20ec9cc24d6e?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

**Review by grahamsmith-okta — APPROVED** [review]

**Pull request merged**